packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -20,13 +20,18 @@ import {
   stripPort,
 } from './ManifestHandler';
 
-function getPlatformFromRequest(req: express.Request | http.IncomingMessage): string {
+function getPlatformFromRequest(req: express.Request | http.IncomingMessage): 'android' | 'ios' {
   const url = req.url ? parse(req.url, /* parseQueryString */ true) : null;
   const platform = url?.query.platform || req.headers['expo-platform'];
   if (!platform) {
     throw new Error('Must specify expo-platform header or query parameter');
   }
-  return String(platform);
+
+  const stringifiedPlatform = String(platform);
+  if (!['android', 'ios'].includes(stringifiedPlatform)) {
+    throw new Error(`platform must be "android" or "ios". Recieved: "${platform}"`);
```

Comment:
Web is not supported for now: https://exponent-internal.slack.com/archives/C013ZK4SA12/p1630699514008700

Reply:
(please correct me if I'm misunderstanding this...)

Reply:
talking with @wschurman in general this endpoint is in development so very low risk.